daffodil-test/src/test/resources/org/apache/daffodil/section00/general/disallowDocTypes.tdml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is an important file for review. There are tests here for loading of the various XML and schemas with Doctypes in them.